modules/op-using-pipelines-as-code-with-bitbucket-cloud.adoc

```diff
@@ -0,0 +1,199 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
```

Comment:
Global comment-- Please check all through this assembly. This is a very long procedure with loads of nested steps. We might need to break the procedure further and provide the information in small chunks.

Reply:
> Global comment-- Please check all through this assembly. This is a very long procedure with loads of nested steps. We might need to break the procedure further and provide the information in small chunks.

Hi @shipsing As @sounix000 suggested this in his comments. I believe we can create a separate Jira for this comment to get it addressed, since this may involve going through the SMEs and QEs again. Do you agree with this approach?

Reply:
Yes @gtrivedi88. We can create a separate JIRA for the same.
I have gone through this assembly. We need to restructure the following three procedures:

- Using Pipelines as Code with GitHub Webhook
- Using Pipelines as Code with GitLab
- Using Pipelines as Code with Bitbucket Cloud

I can see in all these procedures, we have the scope to segragate them in 2 sub-procedures:

- One to create a CR automatically
- Other to create a CR manually

Thanks

Reply:
> Yes @gtrivedi88. We can create a separate JIRA for the same. I have gone through this assembly. We need to restructure the following three procedures:
> 
> * Using Pipelines as Code with GitHub Webhook
> * Using Pipelines as Code with GitLab
> * Using Pipelines as Code with Bitbucket Cloud
> 
> I can see in all these procedures, we have the scope to segragate them in 2 sub-procedures:
> 
> * One to create a CR automatically
> * Other to create a CR manually
> 
> Thanks

You're awesome!
Thank you so much. Will tag a Jira to this comment when that is created.

Reply:
https://issues.redhat.com/browse/RHDEVDOCS-5075